cms/envs/common.py

```diff
@@ -1427,5 +1427,10 @@
 # Queue to use for updating persistent grades
 RECALCULATE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
 
+####################### Persistent Social Engagement ##############################
+
+# Queue to use for updating persistent social engagements
+RECALCULATE_SOCIAL_ENGAGEMENT_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
I guess `cms/envs/aws.py` also needs an update to allow it to be overridden.

Reply:
I see that `LOW_PRIORITY_QUEUE` is a bit different for CMS - `edx.cms.core.low` and I'm wondering if this would work properly.

I was basing here on `RECALCULATE_GRADES_ROUTING_KEY`, which is set only in `lms/envs/aws.py`.